test/acceptance/team_test.rb

```diff
@@ -37,4 +37,32 @@ def test_managing_a_team_as_sole_manager
       end
     end
   end
+
+  def test_team_pagination
+
+    100.times do |index|
```

Comment:
> Our question at this point is how do you add activity

I think you would need to create a bunch of different `UserExercise` records for different exercises all in the same language, and each one would need one submission.